lib/mr_bump/slack.rb

```diff
@@ -16,31 +21,85 @@ def bump(version, changes)
       options = {}
       options[:icon_url] = @icon if @icon
       options[:attachments] = [attatchment(version, changes)]
-      notifier.ping ' ', options
+      notifier.ping 'There is a new version bump!', options
```

Comment:
Yeah I agree that it dosen't add much, but this is the text that comes in with push notifications, desktop, phone etc. If its blank then it just says Mr Bump and its a bit strange. Open to suggestions of what to change it to.